content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
+## API and Terraform Enterprise Provider
+
+In addition to the Terraform Cloud UI, you can use the following methods to manage organizations:
+- [Organizations API](/cloud-docs/api-docs/organizations)
+- The `tfe` provider [`tfe_organization`](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/organization) resource
 
 ## Selecting Organizations
 
-On most pages within Terraform Cloud, the top navigation bar displays the name of the selected organization. Clicking the name reveals the organization switcher menu, which lists all of the organizations you belong to. You can switch to another organization by clicking its name, or you can create a new organization with the "Create new organization" button.
+Terraform Cloud displays your current organization in the left of the top navigation bar. To select an organization:
 
-![screenshot: the organization switcher menu](/img/docs/org-nav.png)
+1. Click the current organization name to view a list of all the organizations where you are a member.
+1. Click an organization to select it. Terraform Cloud displays list of workspaces within that organization.
 
 ## Joining and Leaving Organizations
 
-To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. See [Organization Settings: Users](#users) below for details.
+To join an organization, you must be invited by one of its [owners][] and must accept the emailed invitation. Refer to [Organization Settings: Users](#users) for details.
 
 [permissions-citation]: #intentionally-unused---keep-for-maintainers
 
-You can leave an organization from your user account settings. See [User Settings: Organizations](/cloud-docs/users-teams-organizations/users#organizations) for details.
+You can leave an organization from your user account settings. Refer to [User Settings: Organizations](/cloud-docs/users-teams-organizations/users#organizations) for details.
 
 ## Creating Organizations
 
-Users can create new organizations by clicking the "Create new organization" button in the organization switcher menu. The new organization page also displays automatically when the currently logged-in user does not belong to any organizations, as when first logging in as a new user.
-
-![screenshot: the new organization page](/img/docs/org-new.png)
+On Terraform Enterprise, administrators can restrict your ability to create organizations. Refer to [Administration: General Settings](/enterprise/admin/application/general#organization-creation) for details.
 
-To create a new organization, provide a unique name and a primary contact email address. Organization names can include numbers, letters, underscores (`_`), and hyphens (`-`).
+On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. Otherwise, to create an organization:
```

Comment:
```suggestion
On Terraform Cloud, any user can create a new organization. If you do not belong to any organizations, Terraform Cloud prompts you to create one the first time you log in. To create an organization:
```
It might just be me, but I can't see the two clauses that 'otherwise' is linking here.

Reply:
yeah this is fair. I think I was just trying to be like "you will do this automatically if you don't already have an org" but I feel like it was poorly done. Good edit!